src/main/java/org/folio/dew/batch/acquisitions/edifact/services/SaveToFTPStorageService.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.dew.batch.acquisitions.edifact.services;
+
+import org.folio.dew.domain.dto.EdiFtp;
+import org.folio.dew.domain.dto.VendorEdiOrdersExportConfig;
+import org.folio.dew.repository.FTPObjectStorageRepository;
+import org.folio.dew.repository.SFTPObjectStorageRepository;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@RequiredArgsConstructor
+@Service
+@Log4j2
+public class SaveToFTPStorageService {
```

Comment:
I think better to rename as FTPStorageService because class name is noun, and methods should be as vers